### ISSUE TITLE
Any chance of switching to bottles or Lutris?

### ISSUE THREAD

**Issue by MichaelGame-Dev** [Streamerbot/sb-linux-installer#8]
Lutris and Bottles have become the standard way to run windows apps outside of Steam.

https://usebottles.com/
https://lutris.net/

Personally, given bottles is a flatpak app, I think that would be the best option. Is there any chance of this?

**Comment by Whipstickgostop** [comment]
We won't be "switching", but I may add additional configs, such as a Lutris yaml file. 
Bottles does not allow easily importing custom install scripts as of yet.
Lutris is also (optionally) available on flathub btw :)

**Comment by MichaelGame-Dev** [comment]
> We won't be "switching", but I may add additional configs, such as a Lutris yaml file. Bottles does not allow easily importing custom install scripts as of yet. Lutris is also (optionally) available on flathub btw :)

Lutris would work perfectly fine for me too!

**Comment by Whipstickgostop** [comment]
An initial Lutris install script has been added to the repo at lutris/streamerbot.yaml :)

I've not heavily tested Streamer.bot features with it, and it does throw a UI exception for me when closing the application - but otherwise seems to be running okay?

If you end up using it, I would love any feedback.

**Comment by MichaelGame-Dev** [comment]
> An initial Lutris install script has been added to the repo at lutris/streamerbot.yaml :)
> 
> I've not heavily tested Streamer.bot features with it, and it does throw a UI exception for me when closing the application - but otherwise seems to be running okay?
> 
> If you end up using it, I would love any feedback.

I tried it last weekend and couldn't quite get it to work out. I did some updates. Tried it this morning, it seems to be working just fine now. I need to reboot here at some point and make sure it doesn't break then. I also managed to tweak the Wine registry to set the DPI higher so I could read the text.

I haven't tried to connect it to my twitch chat yet, but I am going to look at the docs and see if I can do so sometime soon. Thanks for your help here. Only thing I see currently is like you mentioned, an error on exit.

I'll also have to see if there's any special stuff I need to update streamer.bot in Linux.
